### ISSUE TITLE
Design database architecture

### ISSUE THREAD

**Issue by radicalsauce** [code-duel/code-duel#11]

**Comment by radicalsauce** [comment]
Hi @allenjprice ;)

**Comment by allenjprice** [comment]
Sorry for the notifications - I was demoing GitHub issues for our
greenfield project kickoff today :)

On Wednesday, February 17, 2016, Kelly Hale notifications@github.com
wrote:

> Hi @allenjprice https://github.com/allenjprice ;)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/code-duel/code-duel/issues/11#issuecomment-185431918.

## 

Allen J Price https://plus.google.com/u/0/+AllenJPrice/posts
501.593.0870

**Comment by Morgantheplant** [comment]
Well hello @radicalsauce @allenjprice! Hope all's well!

**Comment by radicalsauce** [comment]
@Morgantheplant things are great, how are things with you?

**Comment by Morgantheplant** [comment]
@radicalsauce goin great! We should all grab a beer sometime! I was a big dork and missed the reunions so it would be great to catch up.

**Comment by radicalsauce** [comment]
@Morgantheplant I agree. I was super sick during our 1 year reunion and I was out hiking the PCT during HRX. :disappointed: Miss you guys!
